contrib/holland-commvault/holland_commvault/commvault.py

```diff
@@ -41,6 +41,13 @@ def main():
 
     argv = sys.argv[1:]
 
+    holland_conf = ''
```

Comment:
I didn't fix commvault on purpose as it is likely not used outside of specific environments. If @abg wants it fixed as well though I can.